Graphics/GraphicsEngineOpenGL/src/GLContextLinux.cpp

```diff
@@ -172,11 +172,6 @@ void GLContext::SwapBuffers(int SwapInterval)
             glXSwapIntervalMESA(SwapInterval);
         }
 #    endif
```

Comment:
Looks like `GLX_MESA_swap_control` should not be nested into `GLX_EXT_swap_control`:

```cpp
#if GLX_EXT_swap_control
        if (glXSwapIntervalEXT != nullptr)
        {
            glXSwapIntervalEXT(display, wnd, SwapInterval);
        }
#elif GLX_MESA_swap_control
        if (glXSwapIntervalMESA != nullptr)
        {
            glXSwapIntervalMESA(SwapInterval);
        }
#endif
```

Could you please also merge the changes into one commit to not clobber the history?

Reply:
With this it's never going to use glXSwapIntervalMESA, because GLX_EXT_swap_control is always enabled.

Reply:
Ah, Ok. I now see that you version is correct while my is not. Can you merge the history?